openml/evaluations/functions.py

```diff
@@ -153,6 +160,8 @@ def _list_evaluations(
         api_call += "/flow/%s" % ','.join([str(int(i)) for i in flow])
     if uploader is not None:
         api_call += "/uploader/%s" % ','.join([str(int(i)) for i in uploader])
+    if study is not None:
```

Comment:
Could you please add a unit test to `tests.test_evaluations.test_evaluation_functions` following the pattern there?

Reply:
As we can not assume that after resetting of the server there are any runs. For this reason, I can not hard-code any run ids in a unit test. 

I added this functionality to the study unit test, so this functionality will get tested every time on a newly created study.

Reply:
line 157 of test_study_functions.py